CONTRIBUTING.adoc

```diff
@@ -50,101 +45,98 @@ https://github.com/openshift/origin[origin repo].
 === Develop locally on your host
 
 You can develop OpenShift 3 on Windows, Mac, or Linux, but you'll need Docker
-installed on Linux to actually launch containers.
+installed on Linux to actually launch containers. Client and server binaries
+can be built locally or in the `openshift/release` container environemnt. The
+http://golang.org/[Go] programming language is only necessary for building on
+the local host.
```

Comment:
> I have only added it in f26 and up.

@jcajka it comes by default from the Go website -- why choose to leave it out in the RPM?

> test by some env var/ flag to buildscripts?

Yes, we have a flag to turn it off -- @adelton FYI: `$DETECT_RACES=false`